tools/packaging/rpm/Dockerfile.build

```diff
@@ -1,22 +1,36 @@
 FROM centos:7
 
-RUN yum upgrade -y && \
-    yum install -y epel-release centos-release-scl && \
-    yum install -y fedpkg sudo make which cmake3 rh-git218 \
-                   automake autoconf autogen libtool ninja-build \
-                   llvm-toolset-7 devtoolset-7-libatomic-devel && \
+RUN yum install -y centos-release-scl epel-release && \
+    yum update -y && \
+    yum install -y fedpkg sudo devtoolset-7-gcc devtoolset-7-gcc-c++ \
+                   devtoolset-7-binutils java-1.8.0-openjdk-headless rsync \
+                   rh-git218 wget unzip which make cmake3 patch ninja-build \
+                   devtoolset-7-libatomic-devel openssl python27 libtool autoconf && \
     yum clean all
 
 # Install go (go package in yum repositories is too old)
-RUN curl -o /root/go.tar.gz https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz && \
+RUN curl -o /root/go.tar.gz https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz && \
     tar zxf /root/go.tar.gz -C /usr/local
 ENV GOROOT=/usr/local/go \
-    PATH=/usr/local/go/bin:${PATH}
+    PATH=/usr/local/go/bin:/opt/rh/rh-git218/root/usr/bin:/opt/rh/devtoolset-7/root/usr/bin:/opt/llvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${PATH}
```

Comment:
Not sure if those are actually needed, since the build scripts do something similar.

Reply:
It is required for building envoy on CentOS-7.

see https://github.com/envoyproxy/envoy-build-tools/blob/master/build_container/Dockerfile-centos